assets/src/service/client.js

```diff
@@ -1,10 +1,12 @@
-import ApolloClient from 'apollo-boost'
+import { ApolloClient, InMemoryCache } from '@apollo/client'
 import Cookie from 'js-cookie'
 
 export default new ApolloClient({
+  uri: 'http://localhost:5001/graphql',
```

Comment:
@jonespm, this needs to be configurable, I guess? Not sure what changed to make this required. Probably would pass the hostname through `globals`. Do we have that in the backend settings somewhere?

Reply:
I don't think so. Could you just make it relative and set it to be `/graphql`? Or maybe use `window.location.origin`?

Reply:
Looks like changing it to `/graphql` was fine.